src/js/themes/base.d.ts

```diff
@@ -0,0 +1,825 @@
+                width: string;
+                radius: string;
+                color: string;
+            };
+        };
+        debounceDelay: number;
+        drop: {
+            background: string;
+            border: {
+                width: string;
+                radius: string;
+            };
+            shadowSize: string;
+            zIndex: string;
+        };
+        edgeSize: {
+            none: string;
+            hair: string;
+            xxsmall: string;
+            xsmall: string;
+            small: string;
+            medium: string;
+            large: string;
+            xlarge: string;
+            responsiveBreakpoint: string;
+        };
+        elevation: {
+            light: {
+                none: string;
+                xsmall: string;
+                small: string;
+                medium: string;
+                large: string;
+                xlarge: string;
+            };
+            dark: {
+                none: string;
+                xsmall: string;
+                small: string;
+                medium: string;
+                large: string;
+                xlarge: string;
+            };
+        };
+        focus: {
+            border: {
+                color: string;
+            };
+        };
+        font: {
+            size: string;
+            height: string;
+            maxWidth: string;
+        };
+        hover: {
+            background: {
+                dark: {
+                    color: string;
+                    opacity: string;
+                };
+                light: {
+                    color: string;
+                    opacity: string;
+                };
+            };
+            color: {
+                dark: string;
+                light: string;
+            };
+        };
+        input: {
+            padding: string;
+            weight: number;
+        };
+        opacity: {
+            strong: number;
+            medium: number;
+            weak: number;
+        };
+        selected: {
+            background: string;
+            color: string;
+        };
+        spacing: string;
+        size: {
+            xxsmall: string;
+            xsmall: string;
+            small: string;
+            medium: string;
+            large: string;
+            xlarge: string;
+            xxlarge: string;
+            full: string;
+        };
+    };
+    accordion: {
+        icons: {
+            collapse: any;
+            expand: any;
+        };
+    };
+    anchor: {
+        textDecoration: string;
+        fontWeight: number;
+        color: {
+            dark: string;
+            light: string;
+        };
+        hover: {
+            textDecoration: string;
+        };
+    };
+    box: {
+        responsiveBreakpoint: string;
+    };
+    button: {
+        border: {
+            width: string;
+            radius: string;
+        };
+        primary: {};
+        disabled: {
+            opacity: number;
```

Comment:
```suggestion
            opacity: number | string;
```
e.g. `opacity="strong"`
I think it is safe to mention it for all opacity styles in this file.

Reply:
Is there a type defined for opacity somewhere? It might be useful to have if not.

Reply:
you can add one to grommet/src/js/utils/index.d.ts if it's not there.

Reply:
Awesome! Should I use the values from this page (https://v2.grommet.io/rangeselector) and number as well?

Reply:
yep 💯